### ISSUE TITLE
COMPONENT_* errors quote displayName instead of filename

### ISSUE THREAD

**Issue by zspitzer** [cflint/CFLint#469]
rather than **TestComp**, the following lint warnings should quote **veryLongComponentName.cfc**

```Issue
Severity:INFO
Message code:COMPONENT_INVALID_NAME
	File:S:\wwwroot\veryLongComponentName.cfc
	Column:0
	Line:1
		Message:Component name TestComp is not a valid name. Please use CamelCase and start with a capital letter.
		Variable:'null' in function: 
		Expression:<cfcomponent hint="test" displayName="TestComp">
<cffunction name="testFunction" displayname="" access="public" output="false" returntype="void" hint="no close tag">
</cffunction>
</cfcomponent>
Issue
Severity:INFO
Message code:COMPONENT_TOO_WORDY
	File:S:\wwwroot\veryLongComponentName.cfc
	Column:0
	Line:1
		Message:Component name TestComp is too wordy, can you think of a more concise name?
		Variable:'null' in function: 
		Expression:<cfcomponent hint="test" displayName="TestComp">
<cffunction name="testFunction" displayname="" access="public" output="false" returntype="void" hint="no close tag">
</cffunction>
</cfcomponent>
Issue
Severity:INFO
Message code:COMPONENT_TOO_LONG
	File:S:\wwwroot\veryLongComponentName.cfc
	Column:0
	Line:1
		Message:Component name TestComp should be shorter than 15 characters.
		Variable:'null' in function: 
		Expression:<cfcomponent hint="test" displayName="TestComp">
<cffunction name="testFunction" displayname="" access="public" output="false" returntype="void" hint="no close tag">
</cffunction>
</cfcomponent>
```

[veryLongComponentName.cfc.txt](https://github.com/cflint/CFLint/files/1305109/veryLongComponentName.cfc.txt)

**Comment by ryaneberly** [comment]
Thanks @zspitzer .  This is fixed in dev branch.
